### PR TITLE
My Sayings layout fixes

### PR DESCRIPTION
**Pull request by Clstroud**
* Converts the PaginationView to a programmatically defined view for easier IBDesignable usage
* Fixes layout issues between various size classes for My Sayings screen

Simulator Screen Shot - iPhone 11 Pro - 2020-03-16 at 17 58 12
Simulator Screen Shot - iPad Pro (11-inch) - 2020-03-16 at 17 58 56
Simulator Screen Shot - iPad Pro (11-inch) - 2020-03-16 at 17 59 01
Simulator Screen Shot - iPhone 11 Pro - 2020-03-16 at 17 58 07

### CODE REVIEW

File: Vocable/Common/Views/PaginationView.swift [hunk 1]
```diff
@@ -7,3 +7,93 @@
...
+            textLabel.font = .systemFont(ofSize: 22, weight: .bold)
+            return
+        }
+        stackView.spacing = 42
+        textLabel.font = .systemFont(ofSize: 28, weight: .bold)
+    }
+
+    override func prepareForInterfaceBuilder() {
+        super.prepareForInterfaceBuilder()
+        textLabel.text = "Page 1 of 3"
```
Comment (jrtb) [inline Vocable/Common/Views/PaginationView.swift:96]: assuming placeholder

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift [hunk 2]
```diff
@@ -7,15 +7,41 @@
...
+            self.paginationView.textLabel.text = "\(pagingProgress.pageIndex + 1) of \(pagingProgress.pageCount)"
+        }).store(in: &disposables)
+        
+        paginationView.nextPageButton.addTarget(carouselCollectionViewController, action: #selector(CarouselGridCollectionViewController.scrollToNextPage), for: .primaryActionTriggered)
+        paginationView.previousPageButton.addTarget(carouselCollectionViewController, action: #selector(CarouselGridCollectionViewController.scrollToPreviousPage), for: .primaryActionTriggered)
+    }
     
     @IBAction func backToSettings(_ sender: Any) {
         self.navigationController?.popViewController(animated: true)
+
```
Comment (jrtb) [inline Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]: extra space?

**Review by jrtb — APPROVED** [review]
👊💰

File: Vocable/Common/Views/PaginationView.swift (hunk 1 above)
Comment (Clstroud) [reply Vocable/Common/Views/PaginationView.swift:96]: This only gets called by Interface Builder. It's so we have a default value to reason with while working with IBDesignables.

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift (hunk 2 above)
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]:
```suggestion
```

**Pull request merged**